iphone/Classes/TiRootViewController.m

```diff
@@ -508,19 +508,6 @@ - (void)extractKeyboardInfo:(NSDictionary *)userInfo
   }
 }
 
-- (void)placeView:(UIView *)targetView nearTopOfRect:(CGRect)targetRect aboveTop:(BOOL)aboveTop
```

Comment:
This bunch of code aas added as part of #4569. The initial author tested against the following suites:

- KS
- KB tests
- Rotation Tests
- Tab Group Memory Tests
- Email, Contacts, Media (All modal/popover stuff)
- Open Close Focus Blur events
- LiveView restart

Can you check with QE if we still have those tests to run against them?

Reply:
This method was used to manage the toolbar position because it was managed manually. Now we are using native api to manage keyboard toolbar, so no need of this method. The #4569 was a bigger change which has affected many components, so it was needed to test all stuff which you have mentioned. Now testing keyboard related stuffs are sufficient.

Reply:
This method was used to manage the toolbar position because it was managed manually. Now we are using native api to manage keyboard toolbar, so no need of this method. The #4569 was a bigger change which has affected many components, so it was needed to test all stuff which you have mentioned. Now testing keyboard related stuffs are sufficient.